### DISCUSSION TITLE
Questions on the projet

### DISCUSSION THREAD

**Discussion by GuilhermeMEESantos** [JinneGeelen/ML-MoCap#2] [Q&A] [answered]
I am a Masters degree student from Portugal and I am participating in an investigation scholarship about implementing a low-cost open-source Motion capture system to be used in Smart Factories.

I would like to make some questions about the system you have implemented.

How do you synchronize the cameras? Do you sync them with any time of Time Protocol, like Precision TP? Or use external sync?
Have you done any performance tests on the Average Precision or FPS? Could this be used with a Jetson Nano kit instead of a Raspberry Pi one? 

Wishing a continuation of good work,
GuilhermeMEESantos

**Comment by JinneGeelen** [comment, marked as answer]
Dear GuilhermeMEESantos, 

Thank you for your question. It sounds like you are working on an exciting project too! 

For this system, we chose to synchronize the modules based on NTP.  I have done performance tests for the delays between camera modules at the initialization of the recordings because the difference in activation of the camera was the aspect causing most delays and something out of our hands (not directly related to the NTP). The tests resulted in an acceptable (for our application) average difference of about 10ms between the modules. However, this might be an overestimation of the error since we were recording a 60Hz screen.

We have only tested it on Raspberry Pi's, so it will not work directly on a Jetson since we want to share an image of the Raspbian OS, including our software. Though the same principle could work well on a Jetson device, maybe the Jetson can even handle multiple cameras connected to one module? Unfortunately, I have no experience with Jetson, so I won't be able to help you any further there.

Good luck with your project and studies, 

Kind regards, Jinne

**Reply by GuilhermeMEESantos** [reply]
Dear Jinne,

Thank you so much for answering me.

About the synchronization, for my project, we are planning to use PTP, though if we have to use NTP your project could serve as a guide on how to use NTP. 

About the Jetson, it is planned to have one Jetson per camera, though it could, theoretically, achieve a good performance in terms of fps. 

Thank you so much for taking the time to answer me. 

Wish you a good venture and work,

Kind regards, Guilherme
